recipes/snappy/meta.yaml

```diff
@@ -0,0 +1,36 @@
+{% set version = "1.1.3" %}
+
+package:
+  name: snappy
+  version: {{ version }}
+
+source:
+  url: https://github.com/google/snappy/releases/download/1.1.3/snappy-1.1.3.tar.gz
+  fn: snappy-1.1.3.tar.gz
+  md5: 7358c82f133dc77798e4c2062a749b73
+
+requirements:
+  build:
+    - msinttypes  # [win]
+    - libtool     # [unix]
+    - automake    # [unix]
+    - pkg-config  # [unix]
+
+test:
+  commands:
+    - test -e $PREFIX/include/snappy.h   # [unix]
+    - test -e $PREFIX/include/snappy-stubs-public.h  # [unix]
+    - test -e $PREFIX/lib/libsnappy.dylib  # [osx]
+    - test -e $PREFIX/lib/libsnappy.a      # [unix]
+    - test -e $PREFIX/lib/libsnappy.so     # [linux]
```

Comment:
Looking good. Could you please indent the selectors in the test section so they are lined up? Also, we may want to test `dll` and `lib` for Windows.